### DISCUSSION TITLE
Post issue filed notification within slack thread instead of the channel

### DISCUSSION THREAD

**Discussion by lalitindoria** [community/community#38594] [Projects and Issues]
We have most discussions on threads in Slack and end up filing issues when we arrive at some conclusion while we are active on the thread. After the issue is filed, the notification about the issue is sent to the channel as a DM which generates noise for other members that are part of the channel.

We recently shifted from JIRA that does this notification thing well by sending a message to the thread. The channel feels less cluttered and we should be able to get the issue completion notification on the same thread. **Nobody** looks at the replies Github sends to its own notification because nobody is involved in it. However, if the same reply was sent to the thread from where the issue was created, it would add so much value!

**Comment by ossanna16** [comment]
Hi there @lalitindoria and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
